rollup.config.js

```diff
@@ -0,0 +1,46 @@
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import replace from '@rollup/plugin-replace'
```

Comment:
We need to add `replace({ 'process.env.BROWSER': 'false' })` for outputs without browser.